packages/react/src/themes/colorUtils.ts

```diff
@@ -16,3 +21,23 @@ export const extendColorScheme = (
   })
   return result
 }
+
+export function pickValuesFromColorScheme<T extends Partial<ComponentAreaName>>(
+  colorScheme: ColorSchemeMapping,
+  componentAreaSubset: T[],
```

Comment:
maybe just `componentAreas`? I mean, this would be sufficient to understand what this method is about, as well as to read its code